release_creation/snapshot/create.py

```diff
@@ -22,9 +22,9 @@ def _get_local_os() -> str:
 
 def _get_extra_platforms_for_os(_os: str) -> List[str]:
     if _os == "mac":
-        return ['macosx_10_9_x86_64', 'macosx_11_0_arm64']
+        return ['macosx_10_9_x86_64', 'macosx_11_0_arm64', 'macosx_10_10_intel', 'macosx_12_0_arm64']
```

Comment:
Supplementary question. Hardcoding things is always kind of odd. Is the plan to add to these in the future? Is there a more general case we can capture? Is this the design we want?

Reply:
It's definitely odd but I think it's justified since what we're doing _is_ odd. These are platform tags that are used to download distributions of packages for other platforms (i.e. versions of an OS/cpu). There is no definitive list (at least that I can find) so I've been adding them here. We could certainly move this to a configuration file but I'm not sure that adds a ton of value at the moment.

Reply:
Yeah I can buy that for the short and long term. Eventually if there's enough we can move it to a manifest style file that we pipe in. But, we'll have to see just how many of these very odd strings get added up. Thanks for the explanation.